doc/release/upcoming_changes/21468.new_feature.rst

```diff
@@ -0,0 +1,6 @@
+New function `np.show_runtime`
+------------------------------
+
+A new function `np.show_runtime` has been added to display the runtime
+information of the machine in addition to `np.show_config` which displays
+the build-related information.
```

Comment:
Double back-ticks for `np.*` to format as code. If you want a link, you need to use `numpy.*` with single back-ticks. We tried telling sphinx that `np` is an alias for `numpy`, but couldn't get the links to work.
```suggestion
New function ``np.show_runtime``
--------------------------------

A new function `numpy.show_runtime` has been added to display the runtime
information of the machine in addition to `numpy.show_config` which displays
the build-related information.
```

Reply:
Thanks for the info! Learning Sphinx on the way :). Fixed via [3d2fe4e](https://github.com/numpy/numpy/pull/21468/commits/3d2fe4ec796be6ae63c482e4c4f74e3c7867ff1d)